audioread/ffdec.py

```diff
@@ -73,6 +75,18 @@ def run(self):
                 # Stream closed (EOF).
                 break
 
+class WriterThread(threading.Thread):
+    """A thread that writes data to a filehandle
+    """
+    def __init__(self, fh, audio=None):
```

Comment:
The parameter names here could be a little more descriptive. Both parameters are actually file-like objects—so maybe we should just call them `outfile` and `infile` or something?